reindex-new.sh

```diff
@@ -0,0 +1,57 @@
+#! /bin/bash
+source "./indexUtil.sh"
+
+# list of fields whose data we want to copy
+documentFieldList=(
+    "id"
+    "tenant_id"
+    "cons_id"
+    "cons_name.first"
```

Comment:
subfields get populated when the main field does, so as long as the subfield is mapped in the new index, it will get populated.